_content/tour/functions.article

```diff
@@ -146,6 +144,67 @@ exceptional situations and resource management. `defer` is used to schedule a fu
 call to be executed just before the function returns, while `panic` is used to
 trigger a run-time error and unwind the stack.
 
+** Closures
```

Comment:
This is about functions, I don't think I want to talk about closures.  I rather have this as a lesson

Reply:
Let's talk about this before you do anything.  Please close this PR if you agree